evening_session.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'session'
+
+class EveningSession < Session
```

Comment:
I feel this inheritance here is overkill as there are no extra metods or props added, can't 2 objects of session do the same?